src/components/navigation/navigation.css

```diff
@@ -60,3 +60,24 @@
   background-color: var(--LIGHT-GRAY);
   border-radius: 50%;
 }
+
+.hamburger-icon-btn i {
+  font-size: 1rem;
+}
+
+.hide-desktop {
+  display: none;
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
if u have time add media for more screen size

Reply:
okay sure